extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -59,23 +59,104 @@ function createDicomWebApi(dicomWebConfig, userAuthenticationService) {
     supportsReject,
     staticWado,
     singlepart,
+    requestTransferSyntaxUID,
+    omitQuotationForMultipartRequest
   } = dicomWebConfig;
 
   const dicomWebConfigCopy = JSON.parse(JSON.stringify(dicomWebConfig));
 
+  const acceptHeader = ['multipart/related']
```

Comment:
What I thought Bill's comment was (@wayfarer3130 ) was that the configuration should come from the data source not the logic 

like this 

```
{
      friendlyName: 'dcmjs DICOMWeb Server',
      namespace: '@ohif/extension-default.dataSourcesModule.dicomweb',
      sourceName: 'dicomweb',
      configuration: {
        name: 'aws',
     	acceptHeader: 'Blahblahblah',
        wadoUriRoot: 'https://d33do7qe4w26qo.cloudfront.net/dicomweb',
        qidoRoot: 'https://d33do7qe4w26qo.cloudfront.net/dicomweb',
        wadoRoot: 'https://d33do7qe4w26qo.cloudfront.net/dicomweb',

        qidoSupportsIncludeField: false,
        supportsReject: false,
        imageRendering: 'wadors',
        thumbnailRendering: 'wadors',
        enableStudyLazyLoad: true,
        supportsFuzzyMatching: false,
        supportsWildcard: true,
        staticWado: true,
        singlepart: 'bulkdata,video',
        // whether the data source should use retrieveBulkData to grab metadata,
        // and in case of relative path, what would it be relative to, options
        // are in the series level or study level (some servers like series some study)
        bulkDataURI: {
          enabled: true,
          relativeResolution: 'studies',
        },
      },
```

note the      	acceptHeader: 'Blahblahblah',

Reply:
Yes I was thinking on it. But why limiting it to the accept header ? We could also let the user define the whole header including authentification.
Maybe we need both way
- one for regular users that will have options such as existing now and in which we added the requested TS
- a header property for advanced user that would want to build their herder from scratch (will allow to set a custom authorization for each dicom web server as for now we have 1 authorization service that apply for all)

What do you think ?

Reply:
for this PR I beleive the accept header is good enough, and we can do the header property in the next PR

Reply:
OK let's try this way